### PR TITLE
Account for devicePixelRatio when scaling counter widgets

### PR DESCRIPTION
**Pull request by eradman**
## What type of PR is this? 

- [x] Bug Fix

## How is this tested?

- [x] Manually

## Related Tickets & Documents

https://github.com/getredash/redash/issues/6521

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

devicePixelRatio=1

10scale

devicePixelRatio=1.8

18scale

### CODE REVIEW

**Comment by justinclift** [comment]
@myonlylonely As you're familiar with building things locally, would you be ok to try a build with this PR in it and see if it's better?

That way we can avoid having to merge everything to master just for people to test it. :smile:

File: viz-lib/src/visualizations/counter/Renderer.tsx [hunk 1]
```diff
@@ -20,7 +20,8 @@ function getCounterScale(container: any) {
   if (container.closest('.visualization-preview') || container.closest('.ant-tabs-tabpane')) {
     return "60";
   }
-  const fontSize = 12 + container.clientHeight / 5;
+  const fontSize = 12 + (container.clientHeight / window.devicePixelRatio / 5);
+  console.log(fontSize);
```
[the remark this answers is not in the document]
Comment (eradman) [reply viz-lib/src/visualizations/counter/Renderer.tsx]: Yes. Thanks

**Comment by eradman** [comment]
@myonlylonely what steps did you use to build the image? We need to make sure that the frontend code was rebuilt

**Comment by eradman** [comment]
Also, please pull up a the developent console get the value of `window.devicePixelRatio` on a Macbook Pro

**Comment by justinclift** [comment]
> We need to make sure that the frontend code was rebuilt

Yeah, that's a good point. :smile:

**Comment by myonlylonely** [comment]
> @myonlylonely what steps did you use to build the image? We need to make sure that the frontend code was rebuilt

`make build` to build the front end, `make compose_build` to build the Docker image, `make create_database` to init the database, `make up` to start the Docker image.
And yes, I also checked in the Chrome Dev Tool, the compiled JavaScript file contained the changes that I made locally.
image


And `window.devicePixelRatio` the value on my MacBook is `2`.
image

**Comment by eradman** [comment]
Thanks for the screenshots.

I'll close this PR since it did not seem to solve the problem, and I want to keep the history of changes to the counter widget as simple as we can.

Let's return the discussion to the linked isssue, or perhaps start a new thread at https://github.com/getredash/redash/discussions to think through how counter widgets should be rendered.

**Pull request closed without merging**